scripts/install.ts

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env tsx
```

Comment:
We should be able to delete the install, preinstall, postinstall scripts, since all of the logic should now be duplicated into the setup script.